Community/Expire/automation/jsr223/python/community/expire/expire.py

```diff
@@ -8,17 +8,17 @@
 
 Limitations:
     - The expire config metadata string must include the default units when
-    using with Items that are defined with Units of Measure.
+      using with Items that are defined with Units of Measure.
 
     - Adding new Items or changing Items with an expire metadata config requires
-    a reload of this script to regenerate the rule triggers.
+      a reload of this script to regenerate the rule triggers.
```

Comment:
I'm confused. I thought the main point of your additions was to remove this requirement. Do we still need to reload the script if we add or change Items?

Reply:
You need to re-scan the item registry to look for new or removed items. Changed items get updated passively when they receive an update.

I've added a switch item that allows re-scanning the item registry and recreating the `expire` rule so that we have updated triggers.

Reply:
So, if I understand correctly, you do not need to reload the script to regenerate the rule triggers with your changes and this bullet point in the docstrings is now incorrect. Right?

Reply:
Correct, though reloading the script will have a similar effect. The reload done via the item will preserve any running timers.

Reply:
OK, let's add to the docstring:

- Adding new Items or changing Items with an exprire metadata config requires a regeneration of the rules. This can be done by reloading the script file or by commanding the reload_expire_ item Switch Item. The reload_expire_item is a variable defined in configuration.py containing the name of the Item.